src/engraving/libmscore/symnames.cpp

```diff
@@ -6015,6 +6032,23 @@ constexpr const std::array<const char*, size_t(SymId::lastSym) + 1> SymNames::s_
     "Large brace",
     "Larger brace",
 
+    "Combining flag 8 (1024th) below straight",
```

Comment:
Translatabilite of those whouel be needed, now that they became '1st class citizen'

Reply:
Why would that be needed? As far as I know, these names are never exposed in the UI. The only reason that I added them, is to make sure that the lists of names stay consistent. 

By the way, the names of the normal flags are not translated either. If the names for the straight flags need to be translatable, shouldn’t those for the normal ones be too?

Reply:
These names are exposed, at least in the Symbols palette of the master palette. But indeed if the normal flags ain't, these don't need it either, so sorry for the noise.